packages/main/src/CalendarHeader.hbs

```diff
@@ -24,6 +24,7 @@
 		<div
 			id="{{_id}}-btn1"
 			class="ui5-calheader-arrowbtn ui5-calheader-middlebtn"
+			.hidden = "{{_isMonthButtonHidden}}"
```

Comment:
you can use ? symbol, meant for boolean attributes- "?hidden", instead of ".hidden"

When _isMonthButtonHidden is false, the "hidden" attribute would not be present:
```html
<div class="ui5-calheader-arrowbtn ui5-calheader-middlebtn">
```
and when _isMonthButtonHidden is true - the hidden attribute would be set
```html
<div class="ui5-calheader-arrowbtn ui5-calheader-middlebtn" hidden>
```

Otherwise, with ".hidden", which is a syntax for setting properties, you would have 
hidden="false" and "hidden="true" and at the end it is possible to have in both cases the "hidden" attribute set, although with different value.